### PR TITLE
Build it, build it? build it; build it! 👷‍♀️🚧🏗(app scaffolding)

### PR DESCRIPTION
**Pull request by AllenBW**
satisfies these mocks https://marvelapp.com/prototype/77d4f5b/screen/70827785 cept the red number, but we dont know if that'll stick so 🤷 🐳 

fixes https://projects.engineering.redhat.com/browse/RHCLOUD-8641
fixes https://projects.engineering.redhat.com/browse/RHCLOUD-8642

Screen Shot 2020-08-24 at 1 51 54 PM

### CODE REVIEW

File: src/Routes/TrustTabs/TrustTabs.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                    </Tab>)}
+            </Tabs>
+        </Main>
+    </React.Fragment >;
+};
+
+TrustTabs.displayName = 'insights-tabs';
+TrustTabs.propTypes = {
+    location: PropTypes.object,
+    history: PropTypes.object
```
Comment (LI0131) [inline src/Routes/TrustTabs/TrustTabs.js:76]: I might be crazy, but where are we getting history from here? We could maybe use the `useHistory` hook?

File: src/Utilities/mergeMessages.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+if (program.langPattern) {
+    LANG_PATTERN = `${LANG_DIR}${program.langPattern}`;
+} else {
+    LANG_PATTERN = `${LANG_DIR}/*.json`;
+}
+
+// Merge translated json files (es.json, fr.json, etc) into one object
+// so that they can be merged with the eggregated 'en' object below
+
+const mergedTranslations = globSync(`${rootFolder}${LANG_PATTERN}`)
```
Comment (LI0131) [inline src/Utilities/mergeMessages.js:43]: :godmode:

File: src/Utilities/mergeMessages.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+.reduce((acc, localeObj) => {
+    return { ...acc, ...localeObj };
+}, {});
+
+// Aggregates the default messages that were extracted from the example app's
+// React components via the React Intl Babel plugin. An error will be thrown if
+// there are messages in different components that use the same `id`. The result
+// is a flat collection of `id: message` pairs for the app's default locale.
+
+const defaultMessages = globSync(`${rootFolder}${MESSAGES_PATTERN}`)
```
Comment (LI0131) [inline src/Utilities/mergeMessages.js:60]: :godmode: :godmode:

File: src/Components/Overview/Overview.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        title: intl.formatMessage(messages.youControlWhat),
+        body: intl.formatMessage(messages.youControlWhatBody)
+    }, {
+        title: intl.formatMessage(messages.dataIsEncrypted),
+        body: intl.formatMessage(messages.dataIsEncryptedBody)
+    }, {
+        title: intl.formatMessage(messages.onlyOneUploaded),
+        body: intl.formatMessage(messages.onlyOneUploadedBody)
+    }];
+    const cardBuilder = card => <Card className='ins-c-card'>
```
Comment (LI0131) [inline src/Components/Overview/Overview.js:26]: Maybe throw these in a `Stack` instead of using `900px` to define width?

Comment (LI0131) [reply src/Components/Overview/Overview.js:26]: Or maybe just convert from `px` to `%`? 🤔

File: src/Routes/TrustTabs/TrustTabs.scss [hunk 5]
```diff
@@ -0,0 +1,5 @@
+@import "~@redhat-cloud-services/frontend-components-utilities/files/Utilities/_all.scss";
+
+.ins-c-dashboard-link {
+    margin: var(--pf-global--spacer--lg) 0;
```
Comment (LI0131) [inline src/Routes/TrustTabs/TrustTabs.scss:4]: Define `margin: var(--pf-global--spacer--lg) 0;` as a global scss var in App.scss and use it here and for `ins-c-title`?

File: src/Routes/TrustTabs/TrustTabs.js (hunk 1 above)
Comment (AllenBW) [reply src/Routes/TrustTabs/TrustTabs.js:76]: routerParams() makes it available

Comment (LI0131) [reply src/Routes/TrustTabs/TrustTabs.js:76]: 🤯

File: src/Routes/TrustTabs/TrustTabs.scss (hunk 5 above)
Comment (AllenBW) [reply src/Routes/TrustTabs/TrustTabs.scss:4]: is this cuz maybe we'll be using the same thing elsewhere? 🤔

File: src/Components/Overview/Overview.js (hunk 4 above)
Comment (AllenBW) [reply src/Components/Overview/Overview.js:26]: oooo 70% looks good, feels better <3

File: src/Routes/TrustTabs/TrustTabs.scss (hunk 5 above)
Comment (LI0131) [reply src/Routes/TrustTabs/TrustTabs.scss]: yeah that's what I was thinking 🤷‍♂️ no need if you don't think we gonna be though 😁

Comment (AllenBW) [reply src/Routes/TrustTabs/TrustTabs.scss]: well we already use it twice (button n title spacing), so down for it! great idea 🤗

File: src/Routes/TrustTabs/TrustTabs.js (hunk 1 above)
Comment (AllenBW) [reply src/Routes/TrustTabs/TrustTabs.js:76]: super black magic, could also use the hook i suppose, think routerparams was pre hook availability 🤷

**Pull request merged**